app/components/buttons/filled-button.jsx

```
@@ -26,7 +27,7 @@ export default function FilledButton(props) {
         showIcon = false;
     }
     return(
-        <button className={`text-btn flex border-[--md-sys-color-outline] items-center justify-stretch focus:ring ${props.className}`} role='button' onClick={props.onClick} disabled={props.disabled}>
+        <button className={`text-btn flex border-[--md-sys-color-outline] items-center justify-stretch focus:ring ${props.className}`} role='button' onClick={props.onClick} disabled={props.disabled} type={props.type}>
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)